doc/mgmt/SONiC_OC_FDB_get_support_HLD.md

```diff
@@ -0,0 +1,395 @@
+60          00:00:00:00:00:06   DYNAMIC       Ethernet20          
+```
+
+##### 3.6.2.2.13 show mac address-table interface static interface Ethernet <id>
+This Command provides the no. of static MAC entries matching the particular physical interface port-id present in the system.
+```
+show mac address-table static interface Ethernet 8
+-----------------------------------------------------------
+VLAN         MAC-ADDRESS         TYPE         INTERFACE           
+-----------------------------------------------------------
+30          00:00:00:00:00:03   STATIC        Ethernet8          
+```
+
+##### 3.6.2.2.14 show mac address-table interface dynamic interface Ethernet <id>
+This Command provides the no. of dynamic MAC entries matching the particular physical interface port-id present in the system.
+```
+sonic# show mac address-table dynamic interface Ethernet 12
+-----------------------------------------------------------
+VLAN         MAC-ADDRESS         TYPE         INTERFACE           
+-----------------------------------------------------------
+40          00:00:00:00:00:04   DYNAMIC       Ethernet12          
+```
+
+##### 3.6.2.2.15 show mac address-table interface static interface PortChannel <id>
+This Command provides the no. of static MAC entries matching the particular portchannel-id present in the system.
+```
+sonic# show mac address-table static interface PortChannel 10
+-----------------------------------------------------------
+VLAN         MAC-ADDRESS         TYPE         INTERFACE           
+-----------------------------------------------------------
+99          00:00:00:00:00:99   STATIC        PortChannel10       
+```
+
+##### 3.6.2.2.16 show mac address-table interface dynamic interface PortChannel <id>
+This Command provides the no. of dynamic MAC entries matching the particular portchannel-id present in the system.
+```
+sonic# show mac address-table static interface PortChannel 11
+-----------------------------------------------------------
+VLAN         MAC-ADDRESS         TYPE         INTERFACE           
+-----------------------------------------------------------
+98          00:00:00:00:00:95   DYNAMIC       PortChannel11       
+```
+
+#### 3.6.2.3 Debug Commands
+N/A
+
+### 3.6.3 REST API Support
+#### 3.6.3.1 GET
+##### Get all support for MAC entries
+- 'openconfig-network-instance:network-instances/network-instance={name}/fdb/mac-table/entries'
+
+##### Get MAC entry with MAC address filter
+- 'openconfig-network-instance:network-instances/network-instance={name}/fdb/mac-table/entries/entry={mac-address},{vlan}'
+
+
+
+# 4 Flow Diagrams
+N/A
+
+# 5 Error Handling
+N/A
+
+# 6 Serviceability and Debug
+N/A
+
+# 7 Warm Boot Support
+N/A
+
+# 8 Scalability
+N/A
+
+# 9 Unit Test
+The following test cases will be tested using CLI/REST/GNMI management interfaces.
+#### FDB test cases:
+1) Verify whether "show mac address-table" command dumps all the MAC entries
```

Comment:
Enclose the command itself in backticks rather than double-quotes, so that they render as monospaced text. 

Also, it would be good if you could render the test cases as a table, with two fields: Test Case Name, Description.

Reply:
Hi,
i didn't see static mac address configuration support, which is there in click command. how are we tracking this?
@jeff-yin @ViramShah91 @venkatmahalingam

Reply:
++ @bhavini-gada

Reply:
@bhavini-gada @jeff-yin 
Appreciate your response. it seems to be a miss as static cli was there in pre Buzznik.

Reply:
It seems the requirement for Buzznik was to support Get requests only. We had added the config items for 3.1 planning & is under review with Broadcom team.